### ISSUE TITLE
508 Error in empty grid cell

### ISSUE THREAD

**Issue by slvy** [telerik/kendo-angular#2598]
When there is an empty data cell within a grid, the accessibility tool ANDI will produce the error 'Elements with No Accessible Name'.  Please advise.  Thanks.

**Comment by danielkaradachki** [comment]
Elements with role [cell](https://www.w3.org/TR/wai-aria-1.1/#cell) do not require accessible name. If this is an issue with the tool that you are using then I can suggest to set some content using a [cell template](https://www.telerik.com/kendo-angular-ui/components/grid/columns/templates/#toc-cell-template) for cells with no value.
